resources/istio/charts/kiali/values.yaml

```diff
@@ -57,3 +57,7 @@ credentials:
   # Override the automatically detected Jaeger URL, useful when Jaeger service has no ExternalIPs
   # jaegerURL:
 prometheusAddr: http://monitoring-prometheus.kyma-system:9090
+global:
```

Comment:
I would suggest moving the overrides for the values to the main `istio/values.yaml`, we have all overrides there

Reply:
this added here because of local minikube installation, somehow does not work without this value in chart values.yam file